packages/dx-react-scheduler/src/types/editing/appointment-tooltip.types.ts

```diff
@@ -28,19 +30,41 @@ export namespace AppointmentTooltip {
     contentComponent: React.ComponentType<AppointmentTooltip.ContentProps>;
     /** A component that renders a command button. */
     commandButtonComponent: React.ComponentType<AppointmentTooltip.CommandButtonProps>;
+    /** A component that renders an icon for recurring appointments inside the AppointmentTooltip. */
+    recurringIconComponent: React.ComponentType<object>;
   }
   /** Describes properties passed to a component that renders the tooltip header. */
   export interface HeaderProps {
     /** The appointment’s displayed metadata. */
     appointmentData?: AppointmentModel;
-    /** A React node used to render the tooltip header. */
+    /** Specifies the Open button’s visibility. */
+    showOpenButton: boolean;
+    /** Specifies the Close button’s visibility. */
+    showCloseButton: boolean;
+    /** Specifies the Delete button’s visibility. */
+    showDeleteButton: boolean;
+    /** A command button’s identifier list. */
```

Comment:
It looks like it should be "The command button's...", but if it's just a comment for internal use, it's ok.